### DISCUSSION TITLE
Add a group for maintainers

### DISCUSSION THREAD

**Discussion by ccoVeille** [gofr-dev/gofr#1182] [Ideas]
Hi everyone,

This project is one of the most actively maintained one I'm participating.

Cheers guy

There is one thing that I'm struggling with.

It's when I have to ping you guys, the ones I identified as active maintainers

So the classic dream team @aryanmehrotra @vipul-rawat @srijan-27 @Umang01-hash 

But here is my problem, mentioning you one by one in the huge list GitHub is suggesting is a real pain.

I would like to suggest you to configure something at repository level, that I did for one of my project

https://github.com/jargonLint/jargonLint

Here is what it looks like when mentioning people

![Screenshot_20241109_101712_Firefox.jpg](https://github.com/user-attachments/assets/600bc278-cdd6-4429-ab6f-48c6f330babd)

So maintainers can be notified as a group.

It can be configured via the settings

![Screenshot_20241109_101807_Firefox.jpg](https://github.com/user-attachments/assets/c2bdbc38-378e-4da7-9a6d-4645c7be005e)

**Comment by Umang01-hash** [comment]
Thanks for your suggestion @ccoVeille. Have added this group @gofr-dev/maintainers. 
Feel free to mention it when required.🙂
